index.bs

```diff
@@ -59,13 +59,13 @@ discontinuity in the {{presentedFrames}}).
 Note: A web author could know if a callback is late by checking whether {{expectedDisplayTime}} is equal
 to *now*, as opposed to roughly one v-sync in the future.
 
-The {{VideoFrameRequestCallback}} also provides useful {{VideoFrameMetadata|metadata}} about the video
+The {{VideoFrameRequestCallback}} also provides useful {{VideoFrameCallbackMetadata|metadata}} about the video
 frame that was most recently presented for composition, which can be used for automated metrics analysis.
 
-# VideoFrameMetadata #    {#video-frame-metadata}
+# VideoFrameCallbackMetadata #    {#video-frame-metadata-callback}
```

Comment:
Should it be video-frame-callback-metadata?
Ditto for other links below

Reply:
Yes, thanks for catching it!